Makefile

```diff
@@ -27,3 +30,6 @@ drop: ## Drops database. Recreate with `up` then `admin`
 
 admin: up ## Launches admin site in default browser
 	open http://localhost:8000/wp-admin
```

Comment:
outside the scope of this pr, but if we change to 80, we'll need to remove the port here too